app/src/amazonWebview/java/org/mozilla/focus/webview/FirefoxWebView.kt

```diff
@@ -48,6 +49,17 @@ internal class FirefoxWebView(
     init {
         setOnLongClickListener(linkHandler)
         isLongClickable = true
+        initBackgroundColor()
+    }
+
+    private fun initBackgroundColor() {
+        // The initial navigation overlay is drawn over the WebView. However, on some Echo Show devices [1], the
+        // overlay fade out animation causes graphical glitches, as if the WebView's background color is transparent
+        // (graphical glitches usually occur when parts of the screen have no draw calls on them). We speculatively
+        // fixed this issue by overriding to an opaque default color.
```

Comment:
Since MTV verified this works, the fix isn't speculative anymore.  Suggest updating that sentence.